azkaban-webserver/src/package/conf/azkaban.properties

```diff
@@ -51,3 +51,8 @@ cache.directory=cache
 # JMX stats
 jetty.connector.stats=true
 executor.connector.stats=true
+
+#ConnectorBufferSizes
+headerBufferSize=4096
```

Comment:
Could we make this name to be 
"jetty.connector.*"

This way it is more clear the setting is regarding jetty.connectors,
and also fits with the "jetty.connector.stats" specification above